docs/ruleset_functions/get_zone_supply_return_exhaust_relief_terminal_fan_power_dict.md

```diff
@@ -0,0 +1,85 @@
+                    - Get the terminal unit fan power: `terminal_fan_power = get_fan_object_electric_power(RMI,terminal.fan)`  
+                    - Check if the terminal unit is associated with this zone: `if terminal in zone.terminals:`   
+                        - Add to the primary flow for the zone for this hvac system:  `zone_primary_air_flow = zone_primary_air_flow + primary_air_flow_terminal`  
+                        - Add to the terminal fan power for the zone for this hvac system:  `zone_total_terminal_fan_power = zone_total_terminal_fan_power + terminal_fan_power`  
+                - Get dictionary of the hvac system's fan system's supply, return, exhaust and relief fan powers in kW: `fan_sys_kW = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(RMI,hvac.fan_system)["Fan_Power"])`  
+                - Get the hvac system total supply fan kW, need to get the first element in the list of fan powers:`hvac_sys_total_supply_fan_power = fan_sys_kW[0]` 
+                - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power = fan_sys_kW[1]` 
+                - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power = fan_sys_kW[2]` 
+                - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power = fan_sys_kW[3]`     
+
+                - Apportion the supply kW to this zone based on the terminal unit primary flow for the zone and the total primary terminal unit flow across all zones that this HVAC system serves: `zone_total_supply_fan_power = zone_total_supply_fan_power + (hvac_sys_total_supply_fan_power * (zone_primary_air_flow/total_terminal_air_flow))`    
+                - Apportion the return kW to this zone based on the terminal unit primary flow for the zone and the total primary terminal unit flow across all zones that this HVAC system serves: `zone_total_return_fan_power = zone_total_return_fan_power + (hvac_sys_total_return_fan_power * (zone_primary_air_flow/total_terminal_air_flow))`  
+                - Apportion the exhaust kW to this zone based on the terminal unit primary flow for the zone and the total primary terminal unit flow across all zones that this HVAC system serves: `zone_total_exhaust_fan_power = zone_total_exhaust_fan_power + (hvac_sys_total_exhaust_fan_power * (zone_primary_air_flow/total_terminal_air_flow))`  
+                - Apportion the relief kW to this zone based on the terminal unit primary flow for the zone and the total primary terminal unit flow across all zones that this HVAC system serves: `zone_total_relief_fan_power = zone_total_relief_fan_power + (hvac_sys_total_relief_fan_power * (zone_primary_air_flow/total_terminal_air_flow))`  
+
+            - Else, the HVAC system serves a single zone: `Else:`  
```

Comment:
Wouldn't the logic above work for both single zone and multi-zone systems? Why do we need a separate routine for this case?

Reply:
@mkarpman It would but to save run time (reduce the number of lines of code that need to be run) I separated out the scenarios.